core/src/test/scala/unit/kafka/admin/TopicCommandWithAdminClientTest.scala

```diff
@@ -692,4 +692,11 @@ class TopicCommandWithAdminClientTest extends KafkaServerTestHarness with Loggin
     assertTrue(output.contains(testTopicName))
     assertFalse(output.contains(Topic.GROUP_METADATA_TOPIC_NAME))
   }
+
+  private def verifyTopicDelete(topic: String): Unit = {
```

Comment:
Can we reuse/update existing TestUtils [method](https://github.com/apache/kafka/blob/trunk/core/src/test/scala/unit/kafka/utils/TestUtils.scala#L1053).

Reply:
@omkreddy Good point. Updated. Thank you!